### ISSUE TITLE
Add apoc.coll.extract(coll, 'property')

### ISSUE THREAD

**Issue by adam-cowley** [neo4j-contrib/neo4j-apoc-procedures#1159]
## Feature description
Function to extract a property from a collection of maps, nodes or relationships.
`apoc.coll.extract(coll, 'property')`

This would be a cleaner way for extracting properties from a collection rather than using  pattern comprehension.

**Comment by jexp** [comment]
Fun fact :)
```
apoc.coll.extract(coll, 'property')
[x IN coll | x.property]
```

**Comment by jexp** [comment]
But the function call is easier to read and allows for dynamic properties, although in this particular case...

`[x IN coll | x['property']]`

**Comment by conker84** [comment]
closed via 36ffa27
